reproductions/basic-sqlite/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Prisma development playground",
   "main": "index.ts",
   "scripts": {
+    "prisma": "node -e 'console.log(require.resolve(\"@prisma/cli\"))'",
```

Comment:
This package is not part of Prisma itself, it's merely a template for a local playground used for development. Changing anything here has no effect for users.